dags/sources/bigquery.py

```diff
@@ -38,7 +38,10 @@ class Source:
   def __init__(self, config: Dict[str, Any]):
     try:
       json.loads(config.get("credentials"))
-    except ValueError:
+    except (ValueError, TypeError):
+      # json.loads fails if credentials are not a valid JSON object
+      config["credentials"] = None
+    except TypeError:
```

Comment:
I think this line is residual

Reply:
line 44

Reply:
Thanks, not sure how it got back in there :) This is now removed with the latest commit.